DOCS/man/options.rst

```diff
@@ -3928,6 +3928,8 @@ Input
 ``--input-media-keys=<yes|no>``
     (macOS and Windows only)
     Enable/disable media keys support. Enabled by default (except for libmpv).
+    On other OSes e.g. Linux, media keys are supported if possible and will
+    always be enabled. On those OSes, this option will have no effect.
```

Comment:
Like I said this just makes it worse.
It instantly prompts the question "why not on Linux?" to which the answer is that media keys are different there (as discussed) but *that* is what the documentation should explain.
It should not just state this without explaining the reason.

Reply:
Also "other OSes, e.g. Linux" is too vague. We know exactly which inputs are affected by this: X11 and (presumably) Wayland